doc/source/user_guide/options.rst

```diff
@@ -188,3 +188,55 @@ to provide search functionality for your documentation.
     If you do not set the "use_meilisearch" option,
     ``ansys-sphinx-theme`` uses the default search functionality 
     inherited from the PyData Sphinx Theme.
+
+Cheatsheet
+----------
+
+The theme option cheatsheet provides a quick reference guide for users to easily access 
+and understand the available options in the Sphinx theme.
+
+To enable the cheatsheet, you can add a child 
+dictionary called ``cheatsheet`` to the ``html_theme_options``
+dictionary in the ``conf.py`` file. 
+
+The dict should contain the following keys:
+
+#. ``url``: The URL of the cheatsheet for downloading.
+#. ``title``: Title of the cheatsheet.
+#. ``image``: The thumbnail image for the cheatsheet.
+#. ``local_download``: Specifies whether to download the cheatsheet locally during the build. 
```

Comment:
Can I suggest to use a name more compliant with boolean variables? Something like:

```suggestion
#. ``needs_download``: Specifies whether to download the cheatsheet locally during the build. 
```